app/app.js

```diff
@@ -37,6 +33,23 @@ module.exports = (options) => {
     ...nunjucksOptions // merge any additional options and overwrite defaults above.
   })
 
+  // Cache common component listings
+  const [examples, components, fullPageExamples] = await Promise.all([
+    fileHelper.getDirectories(configPaths.examples),
+    fileHelper.getAllComponents(),
+    fileHelper.getFullPageExamples()
+  ])
```

Comment:
This is good feedback!

Problem came from seeing the app crawl all the component directories, for every page view 😮

What about updating the `gulp serve` (Nodemon) watch list instead?

Then we can still keep things cached until a file is modified

Reply:
As long as it reloads reasonably quickly (so there's minimal chance of reloading and getting a connection error) I'd be happy with that as a solution ⚡

If it takes a while to reload it might be worth living with the 'crawl on every page load' and writing up an issue to revisit in the future? I haven't personally noticed any issues with page load times in the review app – generally feels pretty responsive, even if it is doing lots of file system reads under the hood.

Reply:
Yeah they're just tiny lags

It's the multiple seconds that add up whilst the various tests send requests (100s?) to the review app

I'll try it out and see